examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/speech_client.rb

```diff
@@ -0,0 +1,411 @@
+          #   Overrides the default settings for this call, e.g, timeout,
+          #   retries, etc.
+          # @return [Enumerable<Google::Cloud::Speech::V1::StreamingRecognizeResponse>]
+          #   An enumerable of Google::Cloud::Speech::V1::StreamingRecognizeResponse instances.
+          #
+          # @raise [Google::Gax::GaxError] if the RPC is aborted.
+          #
+          # @note
+          #   EXPERIMENTAL:
+          #     Streaming requests are still undergoing review.
+          #     This method interface might change in the future.
+          #
+          # @example
+          #   require "google/cloud/speech"
+          #
+          #   speech_client = Google::Cloud::Speech.new(version: :v1)
+          #   request = {}
+          #   requests = [request]
+          #   speech_client.streaming_recognize(requests).each do |element|
+          #     # Process element.
+          #   end
+
+          def streaming_recognize reqs, options: nil
+            request_protos = reqs.lazy.map do |req|
+              Google::Gax::to_proto(req, Google::Cloud::Speech::V1::StreamingRecognizeRequest)
+            end
+            @streaming_recognize.call(request_protos, options)
+          end
+
+          ##
+          # Gets the latest state of a long-running operation. Clients can use
+          # this method to poll the operation result at intervals as recommended
+          # by the API service.
+          #
+          # @param name [String]
+          #   The name of the operation resource.
+          # @param options [Google::Gax::CallOptions]
+          #   Overrides the default settings for this call, e.g, timeout,
+          #   retries, etc.
+          # @return [Google::Gax::Operation]
+          # @raise [Google::Gax::GaxError] if the RPC is aborted.
+          # @example
+          #   require "google/cloud/speech"
+          #
+          #   speech_client = Google::Cloud::Speech.new version: :v1
+          #
+          #   op = speech_client.get_operation "-"
+          #
+          #   # Process error operations.
+          #   log_error op.error if op.error?
+          #
+          #   if op.done?
+          #     # Process completed operations.
+          #     log_finished op.response, op.metadata
+          #   else
+          #     # Process pending operations.
+          #     log_pending op.name, op.metadata
+          #   end
+          #
+          def get_operation name, options: nil
+            proto_op = @operations_client.get_operation name, options: options
+
+            Google::Gax::Operation.new(
+              proto_op,
+              @operations_client,
+              Google::Cloud::Speech::V1::LongRunningRecognizeResponse,
+              Google::Cloud::Speech::V1::LongRunningRecognizeMetadata,
+              call_options: options
+            )
+          end
+        end
+      end
+    end
+  end
+end
+
+# Once client is loaded, load helpers.rb if it exists.
+begin
```

Comment:
Do you think we should generate a helpers hook for all clients?

Reply:
I don't know about all clients, but we know for sure Speech will need this.

Reply:
What would be the harm in adding it to all clients?

Reply:
It would add a raise and rescue when each client is required for the first time.